CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+# Contribution Guidelines
+
+Thanks for your interest in contributing to PyCon India 2018 website. Your valuable contribution is much appreciated.
```

Comment:
This should be changed and made `PyCon India Blog`

Reply:
Updated